### PR TITLE
Spelling fixes for doc and README

### PR DESCRIPTION
**Pull request by Floyer007**
Some little fixes to optimize the readability.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,8 @@ Please subscribe and '+1' the videos if you find them useful.
 ## Configuration Tool
 
 To configure Cleanflight you should use the [Cleanflight-configurator GUI tool](https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb
-) (Windows/OSX/Linux) that can be found here:
+) (Windows/OSX/Linux) that can be found [here](https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb
+).
```
Comment (hydra) [inline README.md]: we don't need the link twice.  also 'here' as bad SEO too.  since the link title does not match the content. :smile:

**Comment by hydra** [comment]
will merge once comments are addressed.  can you squash the commits too please?

File: README.md (hunk 1 above)
Comment (Floyer007) [inline README.md]: Oh, yes. You're right :D

**Comment by hydra** [comment]
squashed commits and merged, thanks!

**Pull request closed without merging**